tests/foreman/cli/test_discoveryrule.py

```diff
@@ -240,75 +283,37 @@ def test_positive_create_disabled_rule(self, discoveryrule_factory):
         rule = discoveryrule_factory(options={'enabled': 'false'})
         assert rule.enabled == 'false'
 
-    @pytest.mark.tier3
+    @pytest.mark.tier1
+    @pytest.mark.parametrize('host_name', **parametrized(invalid_hostnames_list()))
     @pytest.mark.parametrize('name', **parametrized(invalid_values_list()))
-    def test_negative_create_with_invalid_name(self, name, discoveryrule_factory):
-        """Create Discovery Rule with invalid names
+    def test_negative_create_rule_with_discovery_params(
+        self, name, host_name, discoveryrule_factory
+    ):
+        """Create Discovery Rule with negative parameters.
 
-        :id: a0350dc9-8f5b-4673-be88-a5e35d1f8ca7
+        :id: 0906cf64-ed0b-49af-844f-1af22f81ab94
 
         :expectedresults: Error should be raised and rule should not be created
 
         :CaseImportance: Medium
```

Comment:
Why `:CaseLevel: Component and  :parametrized: yes` are removed?

Reply:
Might have been removed while code movement